docs/installation.rst

```diff
@@ -192,6 +192,9 @@ license file and another containing the necessary input files for the engine.
 
 .. code-block:: bash
 
-   docker run -it -v /LOCAL/PATH/TO/GUROBI.LIC:/usr/share/gurobi_license -v /LOCAL/PATH/TO/INPUT/DATA:/usr/share/data reisejl bash
+   docker run -it `
+   -v /path/to/gurobi.lic:/usr/share/gurobi_license `
+   -v /path/to/input/data:/mnt/bes/pcm `
```

Comment:
I had some intermediate code changes when I did this that were using the paths from const.py, but I think at this point it can be arbitrary and just has to be consistent with the command line args.

Reply:
Done